n2y/converter.py

```diff
@@ -384,46 +385,73 @@ def to_pandoc(self):
 
 
 class ImageBlock(Block):
+    # `ImageBlock` Class Object Initialization
```

Comment:
Why do you think this comment is worth including?

Reply:
Actually, I'd go through and ask this question about all of the added comments. In general, comments should focus on why the code does what it does and should generally avoid explaining what the code does, since good naming can already do that.

Reply:
With that objective in mind I just deleted them all.